person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age > 18
+  end
```

Comment:
You did well with this project, please remeber that the condition should be true if age is greater than or equal to 18, you only implemented that for when it's greater than 18, so it should be something like this :

`
  private

  def of_age?
    @age >= 18
  end
`